### PR TITLE
2801: Remove shadow in header on mobile

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Remove the shadow in the header on mobile (web).

### Proposed changes

Remove shadow and padding for header and first children on mobile web.

### Side effects

Less padding between the icons (but imo not needed).

### Resolved issues

Fixes: #2801.

---

### CODE REVIEW

File: web/src/components/Header.tsx [hunk 1]
```diff
@@ -57,12 +57,6 @@ const Row = styled.div`
     min-height: ${dimensions.headerHeightSmall}px;
     overflow-x: auto;
     padding: 8px 0;
-    box-shadow: 0 2px 5px -3px rgb(0 0 0 / 20%);
-
-    :first-child {
-      box-shadow: 0 2px 5px -3px rgb(0 0 0 / 12%);
-      padding: 0 4px;
-    }
```
Comment (steffenkleinle) [inline web/src/components/Header.tsx:65]: Does anybody have an idea why this was needed? And why this was not a problem before and only started leading to this weird shadow now?

Comment (f1sh1918) [reply web/src/components/Header.tsx:65]: the answer to that question can be found here:
https://github.com/digitalfabrik/integreat-app/blob/821b2b0fb666f6f90a602fc869c659f8466377f5/web/package.json#L58 @steffenkleinle 
But no idea why sarah added this shadows tbh
At least update of styled components library made this style working

**Review by f1sh1918 — APPROVED** [review]
Tested on chrome & firefox.
Thx for the quick fix

**Review by LeandraH — APPROVED** [review]
Very nice, thanks! Tested in Firefox and Safari (OS)

**Pull request merged**